### PR TITLE
STA-162: Upgrade our React Native tracker to 0.1.2

### PR DESCRIPTION
**Pull request by YMufleh**
Updated globe fork to meet snowplow react-native-0.1.2.

### CODE REVIEW

File: DemoApp/package.json [hunk 1]
```diff
@@ -10,7 +10,11 @@
     "lint": "eslint ."
   },
   "dependencies": {
+<<<<<<< HEAD
```
Comment (max-tgam) [inline DemoApp/package.json:13]: This looks like unresolved Git rebase comment. Needs to be fixed.

File: DemoApp/yarn.lock [hunk 2]
```diff
@@ -987,10 +987,17 @@
     eslint-plugin-react-native "3.6.0"
     prettier "1.16.4"
 
+<<<<<<< HEAD
```
Comment (max-tgam) [inline DemoApp/yarn.lock:990]: This also looks like unresolved Git rebase comment. Needs to be fixed.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,10 @@
 {
   "name": "@snowplow/react-native-tracker",
+<<<<<<< HEAD
```
Comment (max-tgam) [inline package.json:3]: This also looks like unresolved Git rebase comment. Needs to be fixed.

**Review by max-tgam — CHANGES_REQUESTED** [review]
Some unresolved git rebase comments were committed, please resolve those.

**Review by max-tgam — APPROVED** [review]
LGTM

**Comment by max-tgam** [comment]
@abiswas-tgam adding you as an FYI, please take a quick look when you can.

**Review by abiswas-tgam — APPROVED** [review]
Look good to me!!

**Pull request merged**